### PR TITLE
Added docker suport for all

### PR DESCRIPTION
**Pull request by artur99**
Added docker support for all projects + docker-compose file

### CODE REVIEW

File: proj-rust/main.rs [hunk 1]
```diff
@@ -66,8 +66,8 @@ async fn main() {
         .route("/get_first_values", get(get_first_values))
         .with_state(state);
 
-    let addr = SocketAddr::from(([127, 0, 0, 1], 3003));
-    println!("Listening on http://{}", addr);
+    let addr = SocketAddr::from(([0, 0, 0, 0], 3003));
```
Comment (Kavarna) [inline proj-rust/main.rs:69]: Why was this change needed?

Comment (artur99) [reply proj-rust/main.rs:69]: Opening the socket at `127.0.0.1` will only make it accessible on the localhost. For opening it for all outside networks, the socket must listen at `0.0.0.0`.

**Review by Kavarna — APPROVED** [review]
